sonaranalyzer-dotnet/CBDE/LICENSE_THIRD_PARTY.txt

```diff
@@ -0,0 +1,3 @@
+Includes following third-party libraries:
```

Comment:
This was extracted from initial zip file: [cbde-windows.1.0.0.14843.zip](https://repox.jfrog.io/repox/webapp/#/artifacts/browse/tree/General/sonarsource-public-releases/org/sonarsource/cbde/1.0.0.14843/cbde-windows.1.0.0.14843.zip)